ocs_ci/ocs/resources/storage_cluster.py

```
@@ -155,6 +155,143 @@ def verify_osd_tree_schema(ct_pod, deviceset_pvcs):
+
+    Raises:
+        TimeoutExpiredError: If the SC keeps getting recreated.
+    """
+    if sc_ocp is None:
+        sc_ocp = OCP(kind=constants.STORAGECLASS)
+
+    remove_storageclass_from_storageconsumer(sc_name)
+    patch_storageclass_ocs_external_label(sc_name)
+
+    if sc_ocp.is_exist(resource_name=sc_name):
+        log.info(f"Deleting StorageClass {sc_name}")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)